### PR TITLE
Release Note for 1.21.0

### PR DESCRIPTION
**Pull request by chunseoklee**
- Release Note for 1.21.0

Signed-off-by: Chunseok Lee <chunseok.lee@samsung.com>

### CODE REVIEW

**Review by seanshpark — DISMISSED** [review]
LGTM thank you!

File: docs/release/1.21/index.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.. ONE documentation master file, created by
+   sphinx-quickstart on Tue Apr 26 10:18:12 2022.
```
Comment (mhs4670go) [inline docs/release/1.21/index.rst:2]:
```suggestion
   sphinx-quickstart on Wed Sep 06 12:18:12 2022.
```

**Comment by mhs4670go** [comment]
Are you posting debian things as well?

File: docs/release/1.21/release-note_1.21.0.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- Show errors for unrecognized options in `one-optimize`
+- Fix shape inference for `StridedSlice`
+- Fix FuseBatchNormWithTConvPass to support TConv with bias
+- Deprecate `--O1` option in `circle2circle`
+- Support gcc-11
+- Support limited Float16 for kernels constants with dequantization to Float32
+
+## ONE Runtime
+
+### Basic Multimodel nnpakage
```
Comment (glistening) [inline docs/release/1.21/release-note_1.21.0.md:27]:
```suggestion
### Basic Multimodel nnpackage
```

**Review by glistening — APPROVED** [review]
LGTM except for a tiny typo.

**Review by glistening — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by hseok-oh — APPROVED** [review]

**Pull request merged**